airflow/providers/openlineage/provider.yaml

```diff
@@ -77,6 +77,14 @@ config:
         example: "airflow.operators.bash.BashOperator;airflow.operators.python.PythonOperator"
         default: ""
         version_added: 1.1.0
+      opt_in:
+        description: |
+          Set this to true if you want to enable OpenLineage metadata extraction and emission
+          on DAG and/or Task level.
+        type: boolean
+        default: "False"
+        example: ~
+        version_added: 1.6.0
```

Comment:
This version number does not look right?

Reply:
Should I leave it empty? When does this option need to be filled out for the provider release?

Reply:
Oh, it marks from which Airflow version it's compatible with, not Airflow provider, isn't it?

Reply:
For `disabled_for_operators` it's set to `1.1.0`, so `1.6.0` lgtm if it's the next release. I think it's provider version, not Airflow (that, we specify in dependencies section)